jena-fuseki2/jena-fuseki-webapp/src/main/java/org/apache/jena/fuseki/cmd/FusekiCmd.java

```diff
@@ -246,21 +247,26 @@ protected void processModulesAndArgs() {
             }
 
             if ( contains(argFile) ) {
-                String filename = getValue(argFile);
-                log.info("Dataset: in-memory: load file: " + filename);
-                String pathname = filename;
-                if ( filename.startsWith("file:") )
-                    pathname = filename.substring("file:".length());
-                if ( !FileOps.exists(filename) )
-                    throw new CmdException("File not found: " + filename);
-                cmdLineConfig.datasetDescription = "file: "+filename;
+                List<String> filenames = getValues(argFile);
                 // Directly populate the dataset.
                 cmdLineConfig.reset();
                 cmdLineConfig.dsg = DatasetGraphFactory.createTxnMem();
-                Lang language = RDFLanguages.filenameToLang(filename);
-                if ( language == null )
-                    throw new CmdException("Can't guess language for file: " + filename);
-                Txn.executeWrite(cmdLineConfig.dsg, ()->RDFDataMgr.read(cmdLineConfig.dsg, filename));
+                String desc = filenames.size() == 1
+                    ? "file: "+filenames.get(1)
```

Comment:
.get(0)? (Spent the day looking at JS and Python, and on mobile, so I might be confusing langs... if so ignore please)

Reply:
Arrg! `.get(0)` both times (quick fix applied).

The messages need refining as well. Logging works different in the two different server startup sequences ("legacy" - sigh).